### PR TITLE
Importing command from text added.

### PR DESCRIPTION
**Pull request by Grzegorian**
Owocna noc :D

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java [hunk 1]
```diff
@@ -78,4 +85,26 @@ public synchronized String getCurrentCommandString() {
...
+        }.getType();
+        List<Command> all = gson.fromJson(loadedCommands, dataListType);
+        List<DriverCommand> po = new ArrayList<DriverCommand>();
+        for (Command t : all) {
+            if (t.operation.equals("SetPositionCommand")) {
+                po.add(new SetPositionCommand(t.posX, t.posY));
+            } else if (t.operation.equals("OperateToCommand")) {
+                po.add(new OperateToCommand(t.posX, t.posY));
+            }
+        }
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java:107]: Parsing should be extracted (it should not be tangled with command management) to separate class.

File: src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java [hunk 2]
```diff
@@ -78,4 +85,26 @@ public synchronized String getCurrentCommandString() {
...
+        setCurrentCommand(po, "Loaded input");
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java:108]: Can you add an optional command name to the input text.

**Review by kowallus — CHANGES_REQUESTED** [review]
Almost OK.

File: src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java [hunk 3]
```diff
@@ -1,13 +1,20 @@
 package edu.kis.powp.jobs2d.command.manager;
 
-import java.util.Iterator;
-import java.util.List;
-
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java:4]: What about dependencies to gson? I cannot see pom.xml changes

File: src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java (hunk 2 above)
Comment (Grzegorian) [reply src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java:108]: Can you write me an example?

Comment (kowallus) [reply src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java:108]: Something like:
{
"name":  "a point",
"commands": [
  {
    "operation": "SetPositionCommand",
    "posX": -20,
    "posY": -50
  },
  {
    "operation": "OperateToCommand",
    "posX": -20,
    "posY": -50
  }
] 
}

**Comment by kowallus** [comment]
Reopen PR after modifications

**Pull request closed without merging**